### ISSUE TITLE
Kane's standard exclusions for simple discounts added outside rules

### ISSUE THREAD

**Issue by kbower** [kbower/tickettest1#1662]
_Imported from trac ticket 1662.
Created by **wayne**
Opened in trac: May 23, 2012, 9:39:28 AM
Last modified in trac: May 23, 2012, 10:16:02 AM_
***
_by wayne:_
I added a discount code in train as a test to see if it would automatically include the standard exclusions and it did not. Can we also make that change  for discount codes added in Loft?

Thanks,

Wayne Liburd

**Comment by kbower** [comment]
_**kbower** trac comment on May 23, 2012, 9:40:56 AM:_

From LIVE, select current discounts that are not in the current `excluded_check.keys()`:

```
  1  select discountid from discounts where discountid not in ('10K','10S','10T','20T','220','4TH','50D','50G','51D','51G','51P','5PB','5PD','5T','60A','63A','BTC','EFD','FB','IR','LAB','LD','LET','MAT','MOM','MOT','NYS','PDP','PRE','REB','REC','SER','SNR','SOF','STM','TBL','TEN','THI','TPM','TYD')
  2* order by 1
SQL> /

DIS
---
000
1
10
100
10P
13
15
150
1JC
2
20
200
20A
250
2JC
3
30
300
350
3JC
4
40
400
4JC
4JF
5
50
500
54
5JC
6
6.5
60
600
62
7
8
800
9
A10
A20
A60
AMX
B00
B05
B10
B15
B20
B50
B75
BED
BGO
BIL
C1
C10
C11
C12
C13
C14
C15
C16
C17
C18
C19
C2
C20
C21
C22
C23
C24
C3
C4
C5
C6
C7
C8
C9
CHA
CHI
CIT
CSN
ENT
FAM
FFR
FGO
FML
FRI
HAF
HUN
K07
K08
K10
K12
K13
K16
K17
KSK
LD5
LDC
LIN
LOT
LTR
MD1
MD2
MD3
MD4
MD5
ME1
ME2
ME3
ME4
ME5
MEL
MLB
NEW
OSW
PIZ
PM1
PM4
PM8
PRS
RAC
RAY
SAR
SEN
TAX
TIX
TWE
TYB
TYC
WB1
WBR
WEB
Y10

134 rows selected.

```

**Comment by kbower** [comment]
_**kbower** trac comment on May 23, 2012, 10:16:02 AM:_

(Changeset [[2684]](../commit/e4db380238cf68e7c38181b8488912830da75c97))
 * closes #1662 (Allow Kane's to add simple discounts to database that still have standard exclusions)

**Comment by kbower** [comment]
_**kbower** trac status change on May 23, 2012, 10:16:02 AM:_ **closed**
